samples/mobile/src/main/java/com/gluonhq/maps/samples/mobile/MobileSample.java

```diff
@@ -68,7 +70,28 @@
 
     @Override
     public void start(Stage stage) {
-        MapView view = new MapView();
+        TileRetriever osm=new CachedOsmTileRetriever() {
+             public String buildImageUrlString(int zoom, long i, long j) {
+                 return "http://tile.openstreetmap.org/" +zoom+"/"+ i + "/" + j + ".png";
+                    
+             }
+             public String copyright() {
+                 return "Map data © OpenStreetMap contributors, CC-BY-SA. Imagery © OpenStreetMap, for demo only.";
+             }
+        };
+        
+        TileRetriever mapBox=new CachedOsmTileRetriever() {
+            public String buildImageUrlString(int zoom, long i, long j) {
```

Comment:
Add some javadoc, explaining style options for instance (satellite, streets, ...)

Reply:
see comment above